src/main/java/edu/iris/dmc/station/Application.java

```diff
@@ -115,27 +126,17 @@ private void convert(File source, File target, Map<String, String> map)
 			}
 		} else {
 			if (source.length() == 0) {
-				throw new IOException("Couldn't process empty file "+source);
+				throw new IOException("Couldn't process empty file " + source);
 			}
 			MetadataFileFormatConverter<File> converter = null;
-			String extension = null;	
-			String xmlstring = "<FDSNStationXML";
-
-			try (InputStream ts = new FileInputStream(source)) {
-				 Reader r = new InputStreamReader(ts, "US-ASCII");
-				  int i = 0;
-				  String extentionString = "";
-	              int data = r.read();
-	              while(i < 255){
-	              char inputChar = (char) data;
-	              extentionString +=inputChar;
-	              data = r.read();
-	              i = i+1;
-	           } 			
-			if (extentionString.contains(xmlstring)) {
+			String extension = null;
+
+			if (isStationXml(source)) {
+				logger.info("File type is StationXml");
```

Comment:
The name of the format is "StationXML", no code-speak in user-visible messages please.

Reply:
Do we need to add a message to the log the detected file type? This seems like extraneous information.

Reply:
These comments would probably be alright if we change the logger level to DEBUG and change our default logger level to error or warnings.